contributing.md

```diff
@@ -665,6 +665,31 @@ In general, we follow [Google's style guide](https://google.github.io/styleguide
 >
 > `speed *= 0.44704;  // miles per hour to meters per second` : Bad
 
+1. **Accessors must not start with `Get`**
+> Member functions granting read access to protected or private data must look like a noun.
+>
+> `public: ::ServerConfig ServerConfig() const;` : Allowed
+>
+> `public: ServerConfig GetServerConfig() const;` : Not Allowed
+>
+> **Corner Cases**
+>
+> * A class name may conflict with an accessor function. For example, `Model(int)` would conflict with a `Model` class. In these cases, try to follow the `Noun` - `By` pattern. For example:
```

Comment:
The `By` pattern is tough work when you don't have an argument.

I think this can also often be fixed being explicit with namespaces. [For example](https://github.com/ignitionrobotics/ign-gazebo/blob/0996fe0ef676e1f3d237664e3dd6d15a6fb2464a/include/ignition/gazebo/Link.hh#L90-L92):

```
      /// \brief Get the entity which this Link is related to.
      /// \return Link entity.
      public: gazebo::Entity Entity() const;
```

Reply:
The example
> `public: ::ServerConfig ServerConfig() const;` : Allowed

seems to address that, but I can explicitly add this example as an option when the function has no arguments.

Reply:
Done in https://github.com/ignitionrobotics/docs/pull/238/commits/406a77ade975a04397b17c21e6b735b4aa173388